ci/run_test_main.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This is the script used for the run_tests.sh stage in CI
+# for doing unit testing.
+set -ex
+
+# JENKINS-52781 tar function is breaking symlinks
+rm -rf test_results
+mkdir test_results
+# shellcheck disable=SC1091
+source ./.build_vars.sh
+rm -f "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
+mkdir -p "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/"
+ln -s ../../../../../../../../src/control \
+  "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
+DAOS_BASE=${SL_PREFIX%/install*}
+rm -f dnt.*.memcheck.xml vm_test.out nlt-errors.out
+NODE=${NODELIST%%,*}
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+scp -i ci_key "$mydir/run_test_main_node.sh" jenkins@"${nodelist[0]}":/var/tmp
+
+# shellcheck disable=SC2029
+ssh "$SSH_KEY_ARGS" jenkins@"$NODE" "DAOS_BASE=$DAOS_BASE      \
```

Comment:
(lint) The mentioned parser error was in this simple command. [SC1009]
(lint) Couldn't parse this double quoted string. [SC1073]